compositor/sessionmanager/sessionmanager.cpp

```diff
@@ -320,9 +285,6 @@ void SessionManager::requestHybridSleep()
     Q_EMIT hybridSleepRequested();
 }
 
-void SessionManager::cancelShutdownRequest()
-{
-    Q_EMIT shutdownRequestCanceled();
-}
+void SessionManager::cancelShutdownRequest() { Q_EMIT shutdownRequestCanceled(); }
 
-#include "moc_sessionmanager.cpp"
+#include "sessionmanager.moc"
```

Comment:
it's the same, just a different name.
automoc creates moc_*.cpp and ".moc for code with implementation only (not header)

https://cmake.org/cmake/help/v3.0/manual/cmake-qt.7.html#manual:cmake-qt(7)